docs/modules/ROOT/pages/appendix/solutions.adoc

```diff
@@ -3025,4 +3037,17 @@ solution should trigger accordingly.
 
 === Withdraw more ETH from the new wallet than you deposited
 
+. Visit the Digital Wallet from the Orders & Payment Section in the Account dropdown
```

Comment:
Can you mention the endpoint here please?

Reply:
Hey @ShubhamPalriwala have added the necessary links and wikis to some good docs for the attacks mentioned, should be good to go.